elasticsearch-extensions/lib/elasticsearch/extensions/test/profiling.rb

```diff
@@ -72,7 +71,7 @@ def measure(name, options={}, &block)
             result = RubyProf.stop
             # progress.finish
 
-            total = result.threads.reduce(0) { |total,info| total += info.total_time; total }
+            result.threads.reduce(0) { |total,info| total += info.total_time; total }
```

Comment:
Here we have a similar problem: instead of removing the `total` variable, we should probably print it later on where we print the statistics such as `mean` etc.